osbuild/buildroot.py

```diff
@@ -271,6 +271,11 @@ def run(self, argv, monitor, timeout=None, binds=None, readonly_binds=None, extr
             api_path = "/run/osbuild/api/" + api.endpoint
             mounts += ["--bind", api.socket_address, api_path]
 
+        # Bind mount the runner into the container at a well known location
+        runner_name = os.path.basename(self._runner)
+        runner = f"/run/osbuild/runner/{runner_name}"
+        mounts += ["--ro-bind", self._runner, runner]
```

Comment:
Who creates the `runner` directory? Does `bwrap` do that?

Reply:
Yes, in  [`setup_newroot `](https://github.com/containers/bubblewrap/blob/efbcdba68c587be77d614a30d697e99e602285b2/bubblewrap.c#L1182).